src/index.html

```diff
@@ -13,6 +13,47 @@
     />
```

Comment:
There should be an empty line between the <link> tag and the closing </head> tag to improve readability, as they are sibling elements.